maintainers/maintainer-list.nix

```diff
@@ -7090,6 +7090,12 @@
     githubId = 4477729;
     name = "Sergey Mironov";
   };
+  smunix = {
```

Comment:
Please put this into a separate commit with the message ``maintainers: add smunix``.